tests/doctests/build.rs

```diff
@@ -10,58 +10,76 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let mut tests_file = std::fs::File::create(&tests_file_path)?;
 
     let prefix = Path::new(env!("CARGO_MANIFEST_DIR")).join("../..").canonicalize()?;
-    for entry in std::fs::read_dir(prefix.join("docs/language/src"))?
-        .chain(std::fs::read_dir(prefix.join("docs/language/src/concepts"))?)
-        .chain(std::fs::read_dir(prefix.join("docs/language/src/recipes"))?)
-        .chain(std::fs::read_dir(prefix.join("docs"))?)
+    for entry in walkdir::WalkDir::new(&prefix)
+        .follow_links(false)
+        .into_iter()
+        .filter_entry(|entry| entry.file_name() != "target")
     {
         let entry = entry?;
         let path = entry.path();
         if path.extension().map_or(true, |e| e != "md") {
```

Comment:
Just a drive-by thought: perhaps this can be folded into filter_entry?

Reply:
filter_entry also make sure we do not recurse into it.